third_party/gfootball_engine/CMakeLists.txt

```diff
@@ -129,10 +130,10 @@ add_library(gamelib ${GAME_SOURCES} ${GAME_HEADERS})
 add_library(menulib ${MENU_SOURCES} ${MENU_HEADERS})
 add_library(datalib ${DATA_SOURCES} ${DATA_HEADERS})
 
-set(LIBRARIES gamelib menulib datalib blunted2 rt
-   ${Boost_LIBRARIES} ${PYTHON_LIBRARY}
+set(LIBRARIES gamelib menulib datalib blunted2
+   Boost::filesystem Boost::system Boost::thread Boost::${BOOST_PYTHON_VERSION} ${PYTHON_LIBRARY}
    ${SDL2_IMAGE_LIBRARIES} ${SDL2_TTF_LIBRARIES} ${SDL2_GFX_LIBRARIES}
-   ${SDL2_LIBRARY} ${SGE_LIBRARY} ${OPENGL_LIBRARIES})
+   ${SDL2_LIBRARIES} ${OPENGL_LIBRARIES})
```

Comment:
SGE is never really used throughout the project so I removed it. Also the correct target is ${SDL2_LIBRARIES} and not ${SDL2_LIBRARY}